notes/ltc-transfer-penalty.vbs

```diff
@@ -140,71 +137,90 @@ IF transfer_amount <> "" THEN transfer_amount = "$" & transfer_amount
+        Else
+            If (len(baseline_date) < 8 or IsDate(baseline_date) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid baseline date in the MM/DD/YYYY format.  See policy reference at the bottom of the dialog box if you are unsure of how to determined the baseline date."
+            If period_begins = "" then err_msg = err_msg & vbNewLine & "You must enter the start date of the transfer penalty."
+            If (transfer_date = "" or IsDate(transfer_date) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid transfer date."
+            If last_full_month_of_period = "" then err_msg = err_msg & vbNewLine & "You must enter the last full month of the transfer penalty."
+            If (transfer_amount = "" or IsNumeric(transfer_amount) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid transfer penalty amount."
+            If (partial_penalty_amount = "" or IsNumeric(partial_penalty_amount) = False) then err_msg = err_msg & vbNewLine & "You must enter the partial penalty amount, even if the amount is 0."
+            If (date_of_application = "" or IsDate(date_of_application) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid application date."
+            If (date_client_was_otherwise_eligible = "" or IsDate(date_client_was_otherwise_eligible) = FALSE) then err_msg = err_msg & vbNewLine & "You must enter a valid date that the client was otherwise eligible for MA."
+            If waiver_list = "Select one..." then err_msg = err_msg & vbNewLine & "You must select the waiver status."
+            If hardship_waiver_details = "" then err_msg = err_msg & vbNewLine & "You must provide harship waiver details."              
+            If assets_transfered = "" then err_msg = err_msg & vbNewLine & "You must describe the assets transferred."            
+            If case_action = "" then err_msg = err_msg & vbNewLine & "You must case note the case action."
+            If worker_signature = "" then err_msg = err_msg & vbNewLine & "You must sign your case note!"
+            If err_msg <> "" AND left(err_msg, 4) <> "Loop" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine		'error message including instruction on what needs to be fixed from each mandatory field if incorrect
+        End If
     LOOP UNTIL err_msg = ""									'loops until all errors are resolved
 CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
 Loop until are_we_passworded_out = false					'loops until user passwords back in
 
-'ensures that worker has not "passworded" out of MAXIS
+
+'ensures we're in MAXIS, the case is not PRIV and it's in-county. 
 Call check_for_MAXIS(false)
+Call navigate_to_MAXIS_screen_review_PRIV("Case", "Note", is_this_priv)
```

Comment:
All MAXIS entry needs to be uppercased due to 'known issues' in BlueZone...sigh....